bin/docker/dev/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.13-alpine
+
+# Install packages
+RUN apk add --update --no-cache iptables ca-certificates openvpn wireguard-tools bash sudo openresolv gcc musl-dev make linux-headers vim curl tcpdump && rm -rf /var/cache/apk/*
```

No need to clean manually with `--no-cache`
```suggestion
RUN apk add --update --no-cache iptables ca-certificates openvpn wireguard-tools bash sudo openresolv gcc musl-dev make linux-headers vim curl tcpdump
```